Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/realtime/inbound/PassthroughInboundCORE_X12DSRealTime.java

```diff
@@ -42,7 +43,7 @@
      * Default constructor.
      */
```

Comment:
where are the inbound invocations of the auditlogger?  I just see the audit logger added.  Seems to be a mix of designs here, one using abstract inheritance and one not (outbound vs inbound)